wiki/Skinning/Interface/en.md

```diff
@@ -1118,7 +1118,8 @@ Notes:
 Notes:
 
 - This element has no size restrictions.
-- When used in [osu!mania](/wiki/Game_mode/osu!mania), this element is rotated 90 degrees anti-clockwise and is placed at the bottom right of stage.
+- When used in [osu!mania](/wiki/Game_mode/osu!mania), this element is rotated 90 degrees anti-clockwise,
```

Comment:
we don't add line breaks where unnecessary even if they are not rendered -- that is my only complaint

Reply:
Try editing it

Reply:
Done. I'm too used to code and doc conventions enforcing line wraps.